packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -22,17 +22,22 @@ import {
 import {repeat} from 'lit/directives/repeat.js';
 import {styleMap} from 'lit/directives/style-map.js';
 import {live} from 'lit/directives/live.js';
-import Minisearch from 'minisearch';
 import type {Drawer} from '@material/mwc-drawer';
 import {animate, Options as AnimationOptions} from '@lit-labs/motion';
+import algoliasearch, {
+  SearchClient,
+  SearchIndex,
+} from 'algoliasearch/dist/algoliasearch-lite.esm.browser.js';
+import {Task, TaskStatus} from '@lit-labs/task';
+import vars from '../public-vars.js';
```

Comment:
I see `"resolveJsonModule": true` was added to the tsconfig. Is this converting to ts file and importing necessary then?

Reply:
oops forgot to clean up that flag!